Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1330,6 +1330,28 @@ func _vjpSelu<T: TensorFlowFloatingPoint>(
     })
 }
 
+/// Returns a tensor by applying the swish activation function, namely
+/// `x * sigmoid(x)`.
+///
+/// Source: "Searching for Activation Functions" (Ramachandran et al. 2017)
+/// https://arxiv.org/abs/1710.05941
+@inlinable
+@differentiable
+public func swish<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    x * sigmoid(x)
+}
+
+@inlinable
+@derivative(of: swish)
+func _vjpSwish<T: TensorFlowFloatingPoint>(
+    _ x: Tensor<T>
+) -> (value: Tensor<T>, pullback: (Tensor<T>) -> Tensor<T>) {
+    return (swish(x), { v in 
+        let grad = sigmoid(x) * (1.0 + x * (1 - sigmoid(x)))
```

Comment:
```suggestion
        let sigmoid_features = sigmoid(x)
        let grad = sigmoid_features * (1.0 + x * (1 - sigmoid_features))
```

Reply:
I think it does make sense to compute it explicitly once when x tensor is large.